### PR TITLE
[13.0][IMP] sale_order_secondary_unit: Add name to th element + span (improve inheritance).

### PR DESCRIPTION
**Pull request by victoralmau**
Add name to th element + span (improve inheritance).

Please @CarlosRoca13 and @ernestotejeda can you review it?

@Tecnativa TT36255

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This improvement is good anyways.

**Comment by victoralmau** [comment]
> This improvement is good anyways.

Ok, if you think so, let's apply it anyway and reopen this PR.

**Review by CarlosRoca13 — APPROVED** [review]
LGTM

**Review by ernestotejeda — APPROVED** [review]

File: sale_order_secondary_unit/report/sale_report_templates.xml [hunk 1]
```diff
@@ -8,7 +8,9 @@
             expr="//table[hasclass('table','table-sm','o_main_table')]//th[2]"
             position="before"
         >
-            <th class="text-right" groups="uom.group_uom">Second. Qty</th>
+            <th name="th_secondary_uom_id" class="text-right" groups="uom.group_uom">
+                <span>Second. Qty</span>
+            </th>
         </xpath>
         <xpath expr="//span[@t-field='line.product_uom_qty']/.." position="before">
             <td class="text-right" groups="uom.group_uom">
```
Comment (pedrobaeza) [inline sale_order_secondary_unit/report/sale_report_templates.xml]: Add also here the name

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**